apps/simple-trading-app-e2e/src/integration/market-list.test.ts

```diff
@@ -0,0 +1,67 @@
+describe('market list', () => {
+  describe('simple url', () => {
+    beforeEach(() => cy.visit('/markets'));
+
+    it('selected menus', () => {
```

Comment:
chore (not-sure-if-blocking-or-not): i think generally speaking the `it('...')` statements look a lot more readable when we aim to use it expressively, like this could read "it selects menus" when used as `it('selects menus', () => {...})`

Reply:
As far as it's not commonly accepted convention, I think it's just different approach to the names of test cases. I would't change it here, because, to be consequent, I should change it every other cases, and for such act I'm not convinced enough :-)